Northstar.CustomServers/mod/scripts/vscripts/sh_northstar_utils.gnut

```diff
@@ -1,13 +1,7 @@
 globalize_all_functions
 
-// whether the server is a modded, northstar server
-bool function IsNorthstarServer()
-{
-	return GetConVarBool( "ns_is_modded_server" )
-}
```

Comment:
Removing this global function might be potentially bad for mod support? Idk if any mods use it, but if they do then they will break

Reply:
relatively sure nobody does, it's a feature that's never really been fully finished

Reply:
I don't think anyone else uses it either.